approvals/src/test/java/no/sikt/nva/approvals/persistence/DynamoDbRepositoryTest.java

```diff
@@ -0,0 +1,44 @@
+package no.sikt.nva.approvals.persistence;
+
+import static no.sikt.nva.approvals.persistence.DynamoDbConstants.TABLE_NAME;
+import static no.sikt.nva.approvals.persistence.DynamoDbLocal.dynamoDBLocal;
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.junit.jupiter.api.AssertionsKt.assertNull;
+import com.amazonaws.services.dynamodbv2.local.embedded.DynamoDBEmbedded;
+import com.amazonaws.services.dynamodbv2.local.shared.access.AmazonDynamoDBLocal;
+import java.util.List;
+import java.util.UUID;
+import no.sikt.nva.approvals.domain.Approval;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DynamoDbRepositoryTest {
+
+
+    public static final AmazonDynamoDBLocal database = DynamoDBEmbedded.create();
```

Comment:
```suggestion
    public static final AmazonDynamoDBLocal database = DynamoDBEmbedded.create(null, true);
```

This disables telemetry and reduces start-up time by ~500 ms